api/server/router/delta/delta_routes.go

```diff
@@ -0,0 +1,27 @@
+package delta
+
+import (
+	"net/http"
+
+	"github.com/docker/docker/api/server/httputils"
+	"github.com/docker/docker/api/types"
+	"golang.org/x/net/context"
+)
+
+func (d *deltaRouter) postDeltasCreate(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
```

Comment:
Indeed but that's just because the router framework passes it to the handlers. The same pattern can be seen here https://github.com/moby/moby/blob/master/api/server/router/volume/volume_routes.go